Find a functionally equivalent code:android.graphics.drawable.DrawableContainer.getOpticalInsets:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,3 +1,6 @@
 {
-    return (mCurrDrawable == null) ? Insets.NONE : mCurrDrawable.getOpticalInsets();
+    if (mCurrDrawable != null) {
+        return mCurrDrawable.getOpticalInsets();
+    }
+    return Insets.NONE;
 }

